docs/plugins/plugin-structure.md

```diff
@@ -149,3 +150,18 @@ Arguments: `message: Object` – object that you sent from `initializeAsync`
 Use this function to receive data back from your `initializeAsync` function. 
 
 Check `initializeAsync` and `onMessage` [example](./examples.md#using-initializeasync-and-onmessage)
+
+### `settings`
+Type: `Object`
+
+This object is used to specify settings that a plugin user can change. Each setting should include a `description` and a `type`. Other keys include:
+  * `description` -  `String`, description of the setting;
+  * `type` - `String`, used to decide element for rendering a setting.
+    * `string`
+    * `number`
+    * `bool`
+    * `option`
+    * `array`
+  * `defaultValue` - `Any`, default value for the setting;
```

Comment:
1) There is also `label` option to specify label text (it is just a field name by default)
2) I'd notice that `options` and `multi` applicable only for `array` and `option` types
3) For `array` and `option` types also could be added `clearable` and `creatable` options
4) We can add screenshot to show how plugin settings look like
5) We can add a link to react-select configuration to clarify what is creatable, multi, and clearable. Also, to clarify format of options object
6) And the latest step is to add an example plugin with settings to [examples.md](https://github.com/BrainMaestro/cerebro/blob/e67f04d9cd46f4c406643a9e9efd8c287b42c1ac/docs/plugins/examples.md)